nifi-extension-bundles/nifi-py4j-bundle/nifi-py4j-integration-tests/src/test/java/org.apache.nifi.py4j/PythonControllerInteractionIT.java

```diff
@@ -595,6 +595,21 @@ public void testCreateFlowFile() throws IOException {
                 multiLineContent.getBytes(StandardCharsets.UTF_8));
     }
 
+    @Test
+    public void testCreateNothing() throws IOException {
+        /* Test the use-case where the source processor returns with None.
+           (The case must not result in Java NullPointerException.) */
```

Comment:
Probably not necessary to document that it shouldn't throw NPE's, that should always be the case with all code :)

Reply:
True. :D I was too eager to explain why a dummy test processor like that was added.